### PR TITLE
[WFLY-11813] Test for empty username for Elytron's FormAuthenticationMechanism

### PR DESCRIPTION
**Pull request by spriadka**
`FormMechTestCase#testEmptyUsername()` could be added to `FormMechTestBase`, however I'm not sure if it is within desired scope of other test classes extending `FormMechTestBase`

### CODE REVIEW

**Comment by darranl** [comment]
The commit messages should also reference the WFLY Jira issue.

**Comment by spriadka** [comment]
Thank you @mchoma and @darranl. I have squashed the commits and added JIRA reference to commit message.

**Comment by jamezp** [comment]
@darranl or @fjuma could one of you please review this?

File: testsuite/integration/elytron/src/test/java/org/wildfly/test/integration/elytron/http/FormMechTestBase.java [hunk 1]
```diff
@@ -22,45 +22,178 @@
...
+import org.apache.http.client.methods.HttpPost;
+import org.apache.http.client.protocol.HttpClientContext;
+import org.apache.http.impl.client.CloseableHttpClient;
+import org.apache.http.impl.client.HttpClientBuilder;
+import org.apache.http.impl.client.HttpClients;
+import org.apache.http.message.BasicNameValuePair;
+import org.apache.http.util.EntityUtils;
+import org.junit.Test;
+
+import static org.apache.http.HttpStatus.SC_FORBIDDEN;
```
Comment (darranl) [inline testsuite/integration/elytron/src/test/java/org/wildfly/test/integration/elytron/http/FormMechTestBase.java]: The import order is wrong on this one, statics should come first.

**Comment by darranl** [comment]
@spriadka Are you able to get the imports corrected and then we can look to get this merged?

**Comment by bstansberry** [comment]
Resolved via #12744, which just corrected the import order here. Thanks!

**Pull request closed without merging**